src/python/WMCore/Services/SSB/SSB.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+"""
+_SSB_
+
+Service class to be used for fetching site status and metrics
+"""
+
+import logging
+import json
+
+from WMCore.Services.Service import Service
+
+
+class SSB(Service):
```

Comment:
We already have service classes for this.

https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Services/Dashboard/Dashboard.py

I would combine it (either direction - keep SSB and remove that or keep Dashboard)

Reply:
Yes, I wasn't sure about that, but it seems a good idea. Actually, I don't think that Dashboard class is even used. I'll update it, but only tomorrow morning I think :)